extensions/wikia/PortableInfobox/controllers/ApiPortableInfobox.class.php

```diff
@@ -13,12 +13,42 @@ public function execute() {
 		if ( $arguments === null ) {
 			$this->getResult()->setWarning( "Arguments json format incorect or empty" );
 		}
-		$parser = new Parser();
-		$parser->startExternalParse( Title::newFromText( $title ), new ParserOptions(), 'text', true );
-		$frame = $parser->getPreprocessor()->newCustomFrame( $arguments );
+
+		/*
+		$fakeTemplate = 'FakeTemplate' . wfRandomString( 4 );
```

Comment:
I think I would put a note as to why this block of commented code lives here. That way if someone else happens to come across it, they don't just remove it. Other than that though it looks good to me.